### DISCUSSION TITLE
Simple line comment problem

### DISCUSSION THREAD

**Discussion by reubster** [AltimateAI/vscode-dbt-power-user#56] [Q&A] [answered]
Hi, 

Struggling with commenting. 

Jinja comments {# #} in a dbt SQL file seem to cause a syntax error during dbt-run. 
And Jinja comments in a YAML file seem to cause a syntax error during dbt-run. 

In addition, the language interpreter has overwritten both the line comment and the block comment controls. 

It's making dbt power user unusable. 

Any ideas?

**Comment by mdesmet** [comment, marked as answer]
Sorry for the late answer but dbt Power User doesn't do any dbt parsing. If you receive errors when executing dbt run, you're probably using some syntax that is not supported by dbt.
